### PR TITLE
Irods issue 6502 main

### PR DESCRIPTION
**Pull request by JustinKyleJames**
This is to enable Glacier test case.

It is identical to the 4-2-stable except for the one change in the last commit.

### CODE REVIEW

**Comment by JustinKyleJames** [comment]
> Very good. Please squash once confirmed that the test passes

Done.  I did not pound yet.

**Review by alanking — APPROVED** [review]
Let's # it.

**Comment by JustinKyleJames** [comment]
> Let's # it.

I don't know what happened but somehow I lost the commit that was the one line change from 4-2-stable.  I noticed it once I ran tests.

I'm confused but can you take one last look and I will squash and pound.

File: packaging/resource_suite_s3_nocache.py
Comment (korydraughn) [inline packaging/resource_suite_s3_nocache.py:2291]: Is this the line you said disappeared?

**Review by korydraughn — COMMENTED** [review]
Do the tests pass with this version of the PR?

**Review by korydraughn — APPROVED** [review]
Looks good to me.

Let's wait for one more approval before adding the pounds.

File: packaging/resource_suite_s3_nocache.py
Comment (korydraughn) [reply packaging/resource_suite_s3_nocache.py:2291]: Cool. Please mark this conversation as resolved.

**Review by alanking — APPROVED** [review]
Whoops, good catch there...

**Review by korydraughn — APPROVED** [review]
Looks the same to me. Please pound when ready.

**Comment by JustinKyleJames** [comment]
> Looks the same to me. Please pound when ready.

done

**Review by alanking — APPROVED** [review]

**Pull request merged**
